lang/hr/index.md

```diff
@@ -0,0 +1,262 @@
+nešto vrlo slično. Problem je u tome što "slično" nije dovoljno dobro. Bez
+poštivanja neke formalne specifikacije, oznake verzije su u suštini beskorisne
+za upravljanje ovisnostima. Ako damo ime i jasnu definiciju gore navedenim
+idejama, postaje lakše iskomunicirati svoje namjere korisnicima softvera.
+Jednom kada su te namjere jasne i fleksibilne (ali ne previše fleksibilne)
+možemo definirati specifikacije ovisnosti.
+
+Možemo jednostavnim primjerom pokazati kako Semantičko verzioniranje može
+ostaviti pakao ovisnosti u prošlosti. Zamislimo softversku biblioteku (library)
+"Vatrogasno_vozilo." Potreban joj je Semantički verzionirani paket "Ljestve".
+U trenutku kad je "Vatrogasno_vozilo" kreirano, "Ljestve" su na verziji 3.1.0.
+Budući da "Vatrogasno_vozilo" koristi određene funkcionalnosti koje su uvedene
+u verziji 3.1.0, možemo bez straha navesti ovisnost o paketu "Ljestve" verzije
+veće ili jednake 3.1.0, ali manje od 4.0.0. Tako, kada "Ljestve" verzije 3.1.1
+i 3.2.0 budu dostupne, možete ih dodati u svoj sustav upravljanja paketima i
+biti sigurni da će biti kompatibilne s postojećim ovisnim softverom.
+
+Kao odgovoran programer htjet ćete se, naravno, uvjeriti da će svaka nadogradnja
+paketa funkcionirati kao što je navedeno. Stvarni je svijet zbrka i ne možemo
+ništa poduzeti po tom pitanju osim biti oprezni. Ono što možemo napraviti je
+prihvatiti Semantičko verzioniranje kao siguran način za izdavanje i nadogradnju
+paketa bez potrebe za izgradnjom novih verzija ovisnih paketa te tako sačuvati
+vrijeme i živce.
+
+Ako vam ovo sve dobro zvuči, sve što vam je potrebno da bi ste počeli koristiti
+Semantičko verzioniranje je da navedete da ga koristite i slijedite pravila.
+Stavite poveznicu na ovu stranicu u svoju README datoteku, kako bi i ostali bili
+svjesni pravila i mogli imati koristi od njih.
+
+
+Česta pitanja
+---
+
+### Kako upravljati promjenama za inicijalne faze razvoja verzija 0.y.z?
+
+Najjednostavniji je način da započnemo inicijalni razvoj izdavanjem verzije
+0.1.0 te povećavamo oznaku manje verzije za svako sljedeće izdanje.
+
+### Kako znamo kada izdati verziju 1.0.0?
+
+Ako se softver koristi u produkciji, već bi vjerojatno trebao biti na verziji
+1.0.0. Ako već imamo stabilni API, na koji korisnici mogu računati, trebao bi
+biti na verziji 1.0.0. Ako smo zabrinuti zbog kompatibilnosti unatrag, softver
+bi već trebao biti izdan pod verzijom 1.0.0.
+
+### Ne sprječava li ovo ubrzani razvoj i brzo povećavanje verzija?
+
+Veća verzija nula se svodi na brzi razvoj. Ako mijenjamo API svaki dan, trebali
+bi ostati na verziji 0.y.z ili na posebnoj grani raditi na sljedećoj većoj
+verziji.
+
+### Ako i najmanje unatrag inkompatibilne promjene u APIju zahtijevaju povećanje veće verzije, nećemo li vrlo brzo doći do verzije 42.0.0?
+
+Ovo je pitanje odgovornog razvoja i predviđanja. U softver koji ima puno
+ovisnog koda, inkompatibilne promjene potrebno je uvoditi postepeno. Cijena
```

Comment:
"inkompatibilne promjene potrebno je uvoditi postepeno" -> "potrebno je uvoditi postupno inkompatibilne promjene "

Reply:
This would change semantics significantly - the first version explains what you should do when introducing incompatible changes, the second that you **should** include incompatible changes

Reply:
When I read it again, I agree with your comment. Still, I was mostly advocating to change "postepeno" in "postupno".

Reply:
missed that part. agree